### DISCUSSION TITLE
Scrum Prep Milestone #1 - Second Week

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#119] [Weekly Retrospective]
Weekly Development Update: Second week of Milestone 1

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, September 13, 2024 @ 11am.

Thank you for your collaboration!

As for my collaboration:
I took notes on the two meetings held with evand and the team leaders, created a list of lecture topics (#107 ) for everyone to create the lecture topic tasks from, approved issues, gave feedback and created a discussion for how the points will be distributed (#105 )

**Comment by noel-vargas** [comment]
Created a draft for the profile management tab in react native. 

It contains common input fields like name, last name, role, password, phone number etc. And you can upload a profile picture.

**Comment by yadrielsamir** [comment]
Made meeting for the admin interface team where weekly progress was discussed and finished implementing the admins profile design (issue #24)

**Comment by CaribbeanCool** [comment]
I finished my issue regarding the requirements for the map integration (issue #36) and I'm working on the user flow diagram (issue #55)

**Comment by evnd04** [comment]
I was working on the inso metrics, graing issues and verifying the validity of issues submitted from developers, also attending meetings with team leads.

**Comment by lex939** [comment]
This week, I focused on several key tasks to support our milestone progress:

	•	Uploaded 20+ documentation issues, the majority of which the hybrid members will be working on.
	•	Attended both meetings to track the progress of the development teams and to discuss ongoing tasks.
	•	Began preparing the documentation for submission, ensuring everything is aligned for next week’s deadline.

Looking forward to pushing ahead as we move closer to the next milestone.

**Comment by Alejandro-DeJesus** [comment]
Conducted the weekly meeting for the Trust system team, #100, while making progress on #23, hoping to complete the task during the weekend.

**Comment by ghost** [comment]
Met with my team and discussed what the plan would be moving forward. Talked with the databate manager and reviewed PR #112.

**Comment by javier-rivera8** [comment]
I started working on my tasks and researched how to connect to the database. I also ran tests with Expo and began experimenting with it.

**Comment by shenied-maldonado** [comment]
I researched different authentication services and proposed Firebase for our app. With recent approval, implementation can begin soon.

**Comment by JuanJRivera** [comment]
I have been working with the documentation about the task “Documentation of User Login Page Design Process” #48 and working with the task creator Noel Vargas to have the latest information.

**Comment by johnsantiago9** [comment]
I have been researching about the platforms im gonna be using once we finish the sketch design in figma. Got down between 2 designs for the logo and researched about the format and instructions about new documentation issues assigned.

**Comment by joel-alvarado** [comment]
- Designed database schema.
- Wrote SQL scripts for creation of tables.
- Wrote SQL scripts for inserting dummy data.
- Wrote a docker compose config to launch the PostgreSQL container.
- The previous config file can also be configured to run an expo container and a Flask backend.

**Comment by RaquelQuinones** [comment]
Fixed the dependency issues. Researched Node.js and React. Admin login is almost complete.

**Comment by NathBat09** [comment]
Working on issue (#11). Discussed the plan for this milestone with my team and what's its going to be for milestone 2.

**Comment by misaelmercado1** [comment]
Created  Guide for installing react on linux , mac and windows, working on my three documentation tasks.

**Comment by jonVargas1** [comment]
I have been working on issue (#21) and met with my team to discuss organizing our tasks more efficiently.

**Comment by JoseLQV** [comment]
Created the Issue: implement Color Legend for Risk Level Identification in Reports #38 
Completed Issue:  implement Color Legend for Risk Level Identification in Reports #38 

Created Issue: Implement Help Button for Risk Level Color Legend on the Map Feature #133

**Comment by JorgeRivera94** [comment]
Attended team meeting and currently working on issue #12 
Challenges:
* Familiarity with UI design tools. Being mitigated by following online Figma tutorials.

**Comment by Jjulianirizarry24** [comment]
Created issue for initial map view for the Report Map feature. Led and took meeting notes for location management team. Had a busy week so not much progress but next week will be completing the issues assigned.

**Comment by williammartinez10** [comment]
Finished working on user signup and login components, and its corresponding landing page.

**Comment by JorgeC08** [comment]
Working on polishing the draft of personas for the documentation, section 2.2.2

**Comment by Chosen-Juan1** [comment]
This week, I worked on a drawn design for the the admin's interface page. I plan on implementing it on figma should it be approved by the rest of the admin team.

**Comment by lex939** [comment]
The Scrum Prep will start in a few hours. The Weekly Retrospective for the second week of Milestone 1 is now closed. Please note that any replies after this comment will be considered as late replies.
